deploy/crds/crd-challenges.yaml

```diff
@@ -329,6 +329,18 @@ spec:
                             accessKeyID:
                               description: 'The AccessKeyID is used for authentication. If not set we fall-back to using env vars, shared credentials file or AWS Instance metadata see: https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials'
                               type: string
+                            accessKeyIDSecretRef:
+                              description: The SecretAccessKeyID is used for authentication. If not set we fall-back to using env vars, shared credentials file or AWS Instance metadata https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials
+                              type: object
+                              required:
+                                - name
+                              properties:
+                                key:
+                                  description: The key of the entry in the Secret resource's `data` field to be used. Some instances of this field may be defaulted, in others it may be required.
+                                  type: string
```

Comment:
So very curious about the inconsistent use of quotation marks.

Reply:
This was copied from the `secretAccessKeySecretRef` attributes. The quotations all throughout the file seem inconsistent. I think its a good idea! I just think we need to resolve it in another issue entirely so we don't cloud the core feature modifications that this PR presents.